apps/angular/src/app/features/auth/registration/registration.component.html

```diff
@@ -0,0 +1,28 @@
+<form [formGroup]="formGroup" (ngSubmit)="handleSubmit()" class="registration-form">
+    <mat-form-field>
+        <mat-label>First name</mat-label>
+        <input matInput formControlName="firstName">
+    </mat-form-field>
+    <mat-form-field>
+        <mat-label>Last name</mat-label>
+        <input matInput formControlName="lastName">
+    </mat-form-field>
+    <mat-form-field>
+        <mat-label>Email</mat-label>
+        <input type="email" matInput formControlName="email">
+        <mat-error class="registration-form__error" *ngIf="formGroup.controls.email.hasError('email')">{{'Must have an email format'}}</mat-error>
```

Comment:
String interpolation is redundant here we can omit it.
```suggestion
        <mat-error class="registration-form__error" *ngIf="formGroup.controls.email.hasError('email')">Must have an email format</mat-error>
```

Reply:
Please, check this in other components as well. There are several places with similar redundant string interpolation.